src/app/tracing/components/main-tracing.component.html

```diff
@@ -6,19 +6,29 @@
     >
         <mat-sidenav-content class="mat-sidenav-content-container">
             <div id="mainContent">
-                <fcl-graph-empty [hidden]="(isModelLoaded$ | async) === true" />
+                <fcl-graph-empty
+                    [hidden]="
+                        (isModelLoaded$ | async) === true ||
+                        (isModelLoading$ | async) === true
```

Comment:
I don't know whether it is good to hide the graph-empty template on loading, because the loading might fail, then the graph-empty template comes back.